doc/api/axes_api.rst

```diff
@@ -563,7 +563,6 @@ Drawing
    Axes.draw
    Axes.draw_artist
    Axes.redraw_in_frame
-   Axes.get_renderer_cache
```

Comment:
Please revert. Deprecated methods should still appear in the docs until they are finally removed.

The `dreprecated` decorator should add a corresponding message to the docstring. Please check whether this works here because we don't have a docstring. If not, please add a docstring stating the deprecation manually.

Reply:
Good catch, thanks, I must have removed it earlier and forgotten to add it back in. Done now.